inst/rmarkdown/templates/xaringan/resources/rutgers.css

```diff
@@ -0,0 +1,239 @@
+/* 
+CSS for xaringan slides in RU flavor
+Author: Joseph V. Casillas
+Updated: 10/15/2017
+*/
+
+
+/* Package defaults that we slightly tweak */
+
+.inverse {
+  background-color: #272822;
+  color: #777872;
+  text-shadow: 0 0 20px #333;
+}
+.inverse h1, .inverse h2, .inverse h3 {
+  color: #f3f3f3;
+  line-height: 1.0em;
+}
+
+
+
+
+
+
+/*~~~~~~~~~~~~~*/
+/* Title slide */
+/*~~~~~~~~~~~~~*/
+
+/* Add image and change bg color */
+.title-slide {
+  /*background-image: url(http://www.jvcasillas.com/ru_xaringan/img/logo/ru_shield.png);*/
+  background-position: 9% 15%;
+  background-size: 55px;
+  background-color: #fff;
+  padding-left: 100px;  /* delete this for 4:3 aspect ratio */
+}
+
+/* H1 fonts */
+.title-slide h1 {
+  color: #515151;
+  padding-top: 250px;
+  font-weight: normal;
+  font-size: 45px;
+  line-height: 0.05;
+  text-align: left;
+  text-shadow: none;
+  padding-bottom: 18px;
+  margin-bottom: 18px;
+}
+
+/* H2 fonts */
+.title-slide h2 {
+  color: #cc0033;
+  font-size: 35px;
+  text-align: left;
+  text-shadow: none;
+  padding-top: 0px;
+  margin-top: 0px;
+}
+
+/* H3 fonts */
+.title-slide h3 {
+  font-size: 28px;
+  font-weight: normal;
+  color: #666666;
+  text-align: left;
+  text-shadow: none;
+  padding: 0px;
+  margin: 0px;
+  line-height: 1;
+}
+
+/* Remove slide count */
+.title-slide .remark-slide-number {
+  text-shadow: none !important;
```

Comment:
Usually I'm strongly against using `!important` in CSS. Is it possible to remove it? There are a few more instances below.